Kubernetes/omsagent.yaml

```diff
@@ -383,9 +386,7 @@ spec:
         beta.kubernetes.io/os: linux
       # Tolerate a NoSchedule taint on master that ACS Engine sets.
       tolerations:
-        - key: "node-role.kubernetes.io/master"
-          operator: "Equal"
-          value: "true"
+        - operator: "Exists" 
```

Comment:
yet to confirm this with Anders to see if we can include prefernoschedule and noexecute as well.